charts/apisix/values.yaml

```diff
@@ -188,6 +188,8 @@ nginx:
   workerConnections: "10620"
   workerProcesses: auto
   enableCPUAffinity: true
+  # Whether apisix shows the full version on response header Server
+  enableServerTokens: true
```

Comment:
It should not be in the nginx section.

Reply:
hi there, could you point where it should be then?
`apisix` seemed too generic, `gateway` only configures the service

Reply:
@jaysonsantos Just in APISIX.